From: Chelsy Ratnawat <chelsyratnawat2001@gmail.com>
Subject: selftests: fix some typos in tools/testing/selftests
Date: Sat, 3 May 2025 14:19:59 -0700

Fix multiple spelling errors:

 - "rougly" -> "roughly"
 - "fielesystems" -> "filesystems"
 - "Can'" -> "Can't"

Link: https://lkml.kernel.org/r/20250503211959.507815-1-chelsyratnawat2001@gmail.com
Signed-off-by: Chelsy Ratnawat <chelsyratnawat2001@gmail.com>
Cc: Christian Brauner <brauner@kernel.org>
Cc: Shuah Khan <shuah@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 tools/testing/selftests/filesystems/file_stressor.c                      |    2 +-
 tools/testing/selftests/mm/gup_longterm.c                                |    2 +-
 tools/testing/selftests/thermal/intel/power_floor/power_floor_test.c     |    2 +-
 tools/testing/selftests/thermal/intel/workload_hint/workload_hint_test.c |    4 ++--
 4 files changed, 5 insertions(+), 5 deletions(-)

--- a/tools/testing/selftests/filesystems/file_stressor.c~selftests-fix-some-typos-in-tools-testing-selftests
+++ a/tools/testing/selftests/filesystems/file_stressor.c
@@ -156,7 +156,7 @@ TEST_F_TIMEOUT(file_stressor, slab_types
 			ssize_t nr_read;
 
 			/*
-			 * Concurrently read /proc/<pid>/fd/ which rougly does:
+			 * Concurrently read /proc/<pid>/fd/ which roughly does:
 			 *
 			 * f = fget_task_next(p, &fd);
 			 * if (!f)
--- a/tools/testing/selftests/mm/gup_longterm.c~selftests-fix-some-typos-in-tools-testing-selftests
+++ a/tools/testing/selftests/mm/gup_longterm.c
@@ -158,7 +158,7 @@ static void do_test(int fd, size_t size,
 		/*
 		 * R/O pinning or pinning in a private mapping is always
 		 * expected to work. Otherwise, we expect long-term R/W pinning
-		 * to only succeed for special fielesystems.
+		 * to only succeed for special filesystems.
 		 */
 		should_work = !shared || !rw ||
 			      fs_supports_writable_longterm_pinning(fs_type);
--- a/tools/testing/selftests/thermal/intel/power_floor/power_floor_test.c~selftests-fix-some-typos-in-tools-testing-selftests
+++ a/tools/testing/selftests/thermal/intel/power_floor/power_floor_test.c
@@ -56,7 +56,7 @@ int main(int argc, char **argv)
 	}
 
 	if (write(fd, "1\n", 2) < 0) {
-		perror("Can' enable power floor notifications\n");
+		perror("Can't enable power floor notifications\n");
 		exit(1);
 	}
 
--- a/tools/testing/selftests/thermal/intel/workload_hint/workload_hint_test.c~selftests-fix-some-typos-in-tools-testing-selftests
+++ a/tools/testing/selftests/thermal/intel/workload_hint/workload_hint_test.c
@@ -37,7 +37,7 @@ void workload_hint_exit(int signum)
 	}
 
 	if (write(fd, "0\n", 2) < 0) {
-		perror("Can' disable workload hints\n");
+		perror("Can't disable workload hints\n");
 		exit(1);
 	}
 
@@ -99,7 +99,7 @@ int main(int argc, char **argv)
 	}
 
 	if (write(fd, "1\n", 2) < 0) {
-		perror("Can' enable workload hints\n");
+		perror("Can't enable workload hints\n");
 		exit(1);
 	}
 
_
